cfme/tests/generic_objects/test_instances.py

```diff
@@ -284,3 +339,35 @@ def cleanup():
     ).success
 
     assert gen_obj_def_import_export.exists
+
+
+@pytest.mark.meta(automates=[1729341, 1743266])
+def test_generic_object_with_service_button(appliance, generic_object, button_with_dialog):
+    """
+    Bugzilla:
+        1729341
+        1743266
+
+    Polarion:
+        assignee: jdupuy
+        initialEstimate: 1/6h
+        caseimportance: high
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
+        casecomponent: GenericObjects
+    """
+    # add generic object to service
+    myservice = MyService(appliance, name=generic_object.associations.get("services")[0].name)
```

Comment:
Any possibilities of uncaught exceptions with index or attribute?

Reply:
There should be none. Since the generic object is created with the service, it will have that association.